scipy/stats/biasedurn.pyx.templ

```diff
@@ -11,6 +11,21 @@ IF not NPY_OLD:
     from numpy.random cimport bitgen_t
     from cpython.pycapsule cimport PyCapsule_GetPointer, PyCapsule_IsValid
 
+    # If this were C code we could do this:
```

Comment:
@rkern You might be interested in the workaround to use the random C API in Cython using C++.  I could not find any documentation about this limitation and seems to me to be due to the fact that `c_distributions` might only be commonly called from Cython C generated code (as opposed to C++).  I think think this might be due to missing `extern "C"` statements in numpy or could be a bug on Cython's part?

Reply:
I believe this is due to missing `extern "C"` statements, I have a PR open here: https://github.com/numpy/numpy/pull/18264